src/styles.js

```diff
@@ -38,4 +38,14 @@ export const GlobalStyle = createGlobalStyle`
 		bottom: 0;
 		width: 100%
 	}
+	
+	.editButton{
+		position: absolute;
+		top: 20px;
+		right: 10px;
+		border: none;
+		background-color: transparent;
+		color: var(--light);
+		font-size: 0.4rem;
+	}
```

Comment:
I would like to recommend you not to work with css classes, but rather props. This is particularly useful and common for buttons. You have a button component with different variants (default, delete, ....)🤟